test/flask/snaps.spec.ts

```diff
@@ -8,6 +8,10 @@ describe("snaps", function () {
   let metaMaskPage: DappeteerPage;
 
   before(function (this: TestContext) {
+    if (Boolean(process.env.USER_DATA_TEST) === true) {
```

Comment:
probably we can move the check to the helper method as the same code is used multiple times